### PR TITLE
improvement BB-108 ingestion zookeeper alert

### PR DESCRIPTION
**Pull request by naren-rajendran**
Alert for ingestion producer zookeeper operations.

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: monitoring/ingestion/ingestion-producer-alert.yaml [hunk 1]
```diff
@@ -58,14 +58,42 @@ groups:
...
     Labels:
       severity: warning
     Annotations:
       zenko_service: backbeat-ingestion-producer
       description: "Ingestion producer has not queued any messages in the last 5 minutes"
       summary: "Ingestion producer has not queued messages"
+
+  - alert: IngestionProducerZookeeperErrorRate3Percent
+    Expr: |
+      sum(rate(ingestion_zookeeper_operation{status="error", namespace="${namespace}", job="${job_ingestion_producer}"}[5m]))
```
Comment (nicolas2bert) [inline monitoring/ingestion/ingestion-producer-alert.yaml:103]: Same comment about the error/success rate.

**Review by nicolas2bert — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e reset

**Pull request merged**